File: Listas de Exercícios (Python)/Python/Lista 1 - Casa/2.py
print("Conversor: Dólar -> Real\n")

print("--Valores decimais devem ser digitados com (.)!--")

D=float(input("Valor ($): "))

R=D*2.40; R="{:.2f}".format(R)

print(f"\nValor: R${R}\n")

File: Listas de Exercícios (C)/C/Lista 1 - Casa/2.c
#include <stdio.h>

int main()
{
    float R, D;
    
    printf("Conversor: Dólar -> Real\n\n");
    
    printf("--Valores decimais devem ser digitados com (.)!--\n");

    printf("Valor ($): "); scanf("%f", &D);
    
    R=D*2.40;
    
    printf("\nValor: R$%.2f\n", R);
}